website/content/en/troubleshooting/_index.md

```diff
@@ -4,5 +4,68 @@ title: "Troubleshooting"
 linkTitle: "Troubleshooting"
 weight: 4
 description: >
-  Troubleshoot issues with the CSI driver and the Azure Provider
----
\ No newline at end of file
+  An overview of a list of components to assist in troubleshooting.
+---
+
+## Logging
+
+Below is a list of commands you can use to view relevant logs of Azure Key Vault provider and Secrets Store CSI Driver.
+
+### Isolate errors from logs
+
+You can use `grep ^E` and `--since` flag from `kubectl` to isolate any errors occurred after a given duration.
+
+**For Azure Key Vault provider versions less than 0.0.9**
+
+To troubleshoot issues with the csi driver and the provider, you can look at logs from the `secrets-store` container of the csi driver pod running on the same node as your application pod:
+
+```bash
+# find the secrets store csi driver pod running on the same node as your application pod
+kubectl get pods -l app=secrets-store-csi-driver -o wide
+kubectl logs <driver pod name> secrets-store --since=1h | grep ^E
+```
+
+**For Azure Key Vault provider versions equal to and greater than 0.0.9**
+
+For `0.0.9+` the provider logs are available in the provider pods. To troubleshoot issues with the provider, you can look at logs from the provider pod running on the same node as your application pod
+
+```bash
+# find the csi-secrets-store-provider-azure pod running on the same node as your application pod
+kubectl get pods -l app=csi-secrets-store-provider-azure -o wide
+kubectl logs <provider pod name> --since=1h | grep ^E
+```
+
+**For CSI driver logs**
+
+```bash
+# find the secrets store csi driver pod running on the same node as your application pod
+kubectl get pods -l app=secrets-store-csi-driver -o wide
+kubectl logs <driver pod name> secrets-store --since=1h | grep ^E
+```
+
+> It is always a good idea to include relevant logs from Azure Key Vault provider and Secrets Store CSI Driver when opening a new issue.
+
+## Common Issues

```

Comment:
Might be good to add another section for how to uninstall the provider and the driver.